apps/src/p5lab/AnimationTab/AnimationList.jsx

```diff
@@ -9,14 +9,16 @@ import AnimationListItem from './AnimationListItem';
 import NewListItem from './NewListItem';
 import ScrollableList from './ScrollableList';
 import i18n from '@cdo/locale';
-
+import {CURRENT_ANIMATION_TYPE} from '../constants';
 /**
  * Vertical scrolling list of animations associated with the project.
  */
 class AnimationList extends React.Component {
   static propTypes = {
     animationList: shapes.AnimationList.isRequired,
-    selectedAnimation: shapes.AnimationKey,
+    currentAnimations: shapes.CurrentAnimations,
```

Comment:
i think "selected" more accurately describes the purpose of this state. would `selectedAnimations` be appropriate, or do you think `currentAnimations` is a better descriptor now?

Reply:
This property was renamed to distinguish it from a different `selectedAnimations` property which is only used in the AnimationPicker (where students go to add one or multiple new animations from the library to their project).

Reply:
ah, okay, that makes sense then!